presets/rag_service/vector_store/playground/chromadb_playground.py

```diff
@@ -0,0 +1,62 @@
+from llama_index.core import Settings
```

Comment:
Playground can be ignored/removed - currently it is just a single file for testing chromadb index